libs/core/functional/docs/index.rst

```diff
@@ -23,7 +23,7 @@ their arguments.
 * :cpp:func:`hpx::util::deferred_call`
 * :cpp:func:`hpx::invoke`
 * :cpp:func:`hpx::invoke_r`
-* :cpp:func:`hpx::util::invoke_fused`
+* :cpp:func:`hpx::invoke_fused`
```

Comment:
Should we list `hpx::invoke_fused_r` here as well?

Reply:
will do that!